build/Targets/VSL.Imports.targets

```diff
@@ -69,8 +69,9 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == 'DNXCore'">
-      <BaseNuGetRuntimeIdentifier Condition="'$(OS)' == 'Windows_NT'">win7</BaseNuGetRuntimeIdentifier>
-      <BaseNuGetRuntimeIdentifier Condition="'$(OS)' == 'Unix'">ubuntu.14.04</BaseNuGetRuntimeIdentifier>
+    <BaseNuGetRuntimeIdentifier Condition="'$(BaseNuGetRuntimeIdentifier)' == '' and '$(_CoreClrTargetOS)' == 'Windows'">win7</BaseNuGetRuntimeIdentifier>
```

Comment:
Yes, but it matches the surrounding style. I think the default for XML files in VS 2015 is 2 spaces which is why there's a mix of both in these files.